deployment-aws/globus_setup.md

```diff
@@ -0,0 +1,233 @@
+
+First go to [developers.globus.org](developers.globus.org) site and click "Register your app with Globus"):
+![](imgs/p1.png)
+
+This will take you to a page where you can view your Globus projects; add it to the project you used/created when setting up the endpoint. Click the "Add" driopdown and select "Add new app":
+
+![](imgs/p2.png)
+
+Next, enter the information about the application. In particular, ensure the callback URLs are appropriate. These URLs should correspond to the *frontend instance*, NOT the backend. The backend will handle the OAuth2 flow, but is largely orchestrated by the frontend.
+
+TODO: ADD URLS ONCE IMPLEMENTED
+
+![](imgs/p3.png)
+
+
+After this, you should see the following:
+
+![](imgs/p4.png)
+
+**Save the application client ID/secret to your local machine. You will need this for the terraform variables.**
+
+
+
+### Adding the guest collection
+
+Follow the directions at [https://docs.globus.org/how-to/share-files/](https://docs.globus.org/how-to/share-files/); be sure to login using the same identity used to create the endpoint and application. Search for the collection we created earlier; recall the collection was named by the `<DISPLAY NAME>` arg to `globus-connect-server collection create` above. 
+
+Note that in step 5 of the linked instructions, you will select "user" and enter the "email-like" identifier of the web application you created above (e.g. `<web app client id>@clients.auth.globus.org`)
+
+### Exporting the DTN configuration
+
+After setting up the guest share above, we can list the collections associated our endpoint by logging onto the GCS host and running the following (which will likely require a login to localhost as shown above):
+```
+$ globus-connect-server collection list
+ID        | Display Name             | Owner          | Collection Type | Storage Gateway ID                  
+--------- | ------------------------ | -------------- | --------------- | -------------------
+<UUID 1>  | <COLLECTION NAME>        | <GLOBUS OWNER> | mapped          | <GATEWAY UUID 1>
+<UUID 2>  | <SHARED COLLECTION NAME> | <GLOBUS OWNER> | guest           | <GATEWAY UUID 2>
+```
+This is a helpful confirmation that the endpoint and "guest" collection are properly associated.
+
+Now run:
+```
+$ globus-connect-server node create \
+    --client-id "<GCS UUID>" \
+    --export-node node_config.json
+```
+which will prompt for the client secret. The ID and secret here are the ones for the GCS endpoint, *not* the web application. Note that the name of the `--export-node` arg does not matter. However, when we use that file for WebMeV deployments, we expect a name following a particular convention; see [below](#saving-files) for where to persist this file.
```

Comment:
You can use `#saving-the-files` and then no need for the HTML tag in the section heading.
https://stackoverflow.com/questions/2822089/how-to-link-to-part-of-the-same-document-in-markdown